docs/en/compute/Presto.md

```diff
@@ -20,6 +20,22 @@ latency especially when data is remote or network is slow or congested.
 * Table of Contents
 {:toc}
 
+## Using Presto with the Alluxio Catalog Service
+Currently, there are 2 ways to enable Presto to interact with Alluxio:
+* Presto interacts with the [Alluxio Catalog Service]({{ '/en/core-services/Catalog.html' | relativize_url }})
+* Presto interacts with the Hive Metastore (updated with Alluxio locations)
+
+The primary benefits for using Presto with the Alluxio Catalog Service are simpler deployments
+of Alluxio with Presto (no modifications to the Hive Metastore), and enabling schema-aware
+optimizations (transformations like coalescing and file conversions).
+However, currently, the catalog service is limited to read-only workloads.
+
+For more details and instructions on how to use the Alluxio Catalog Service with Presto, please
+visit the [Alluxio Catalog Service documentation]({{ '/en/core-services/Catalog.html' | relativize_url }}).
+
+The rest of this page discusses the alternative approach of Presto interacting with the
+Hive Metastore with updated with Alluxio locations.
```

Comment:
I think this can simply be left at "...interacting with the Hive Metastore."

Reply:
done